### ISSUE TITLE
Smapi android 3.18.3 mods installed but not in launch screen

### ISSUE THREAD

**Issue by Missno** [ZaneYork/SMAPI-Android-Installer#489]
With the latest smapi 3.18.3 it is only loading console commands and the keyboard mods. Even though other mods are in the correct folder and show in the smapi intstaller configure list, when I start up the game it is normal with no errors but the mods just aren't there.
![Screenshot_20230715-001632_My Files](https://github.com/ZaneYork/SMAPI-Android-Installer/assets/139596389/3a180c3e-5762-4f21-a9a3-f1dcaa1fc7e9)

![Screenshot_20230715-001648_SMAPI Installer](https://github.com/ZaneYork/SMAPI-Android-Installer/assets/139596389/b2189639-b2c6-44e9-a553-25648b619208)

![Screenshot_20230715-001734](https://github.com/ZaneYork/SMAPI-Android-Installer/assets/139596389/25272bc0-5b50-4727-9b3e-c25aae2a9f8f)

**Comment by 3Negativ3** [comment]
Did you read the notes? You have to put them into Android/data/com.zane.stardewvalley/files/Mods folder

**Comment by Missno** [comment]
I've tried using Zarchiver to access Android/data/com.zane.stardewvalley/files/Mods and the change the mod path to here but then I'll get the failed to process zip entry virtual keyboard error. I've tried deleting certain files (as other forums said) and reinstalling but still happens. Sorry I'm new to this process, with the previous Smapi I just had to install and just put mods in stardewvalley/mods and it would work.
